manager/manager/deployer_test.go

```diff
@@ -165,7 +165,7 @@ func TestCreateConfiguration(t *testing.T) {
 		defer ts.Close()
 
 		deployer := NewDeployer(ts.URL)
-		err := deployer.CreateConfiguration(valid)
+		_, err := deployer.CreateConfiguration(valid)
```

Comment:
Well, we would actually have to run the kubectl to test this here, or fake it out.